### PR TITLE
182 remove the old doenrolltoken function

### PR DESCRIPTION
**Pull request by lukasmatusiewicz**
Wait some time with merging.

### CODE REVIEW

**Review by nilsbehlen — COMMENTED** [review]
is it not used somewhere in the code anymore?

**Comment by lukasmatusiewicz** [comment]
sorry, didn't check it now.

File: templates/LoginForm.php [hunk 1]
```diff
@@ -231,20 +231,6 @@ class="float-l erroricon" alt="gtk-dialog-error"/>
                                        value="<?php echo htmlspecialchars(@$this->data['imageU2F'] ?: "", ENT_QUOTES) ?>"/>
                                 <input type="hidden" name="imageWebauthn"
                                        value="<?php echo htmlspecialchars(@$this->data['imageWebauthn'] ?: "", ENT_QUOTES) ?>"/>
-
-                                <?php
-                                // If enrollToken load QR Code
-                                if (isset($this->data['tokenQR']))
-                                {
```
Comment (nilsbehlen) [inline templates/LoginForm.php:238]: where will the qr code from enroll via multichallenge be placed?

Comment (lukasmatusiewicz) [reply templates/LoginForm.php:238]: in one of the image inputs. message also comes from the server and this extra fields isn't needed anymore.

**Pull request merged**